content/en/infrastructure/resource_catalog/_index.md

```diff
@@ -30,70 +30,101 @@ Resource Catalog is not available in this site.
+- Enabling Cloud Security Management automatically enables resource collection for the Resource Catalog Inventory tab. Enabling resource collection for the Resource Catalog does _not_ enable the CSM product.
 
 ## Browse the Resource Catalog
 
-On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization. The catalog detects a resource either because it has an Agent installed on it, or because a cloud integration is configured on it. Information about the resources in your organization is shown in the ownership and security tabs, with two views: List and Map.
+On the [Resource Catalog page][2], explore the cloud resources in your Datadog organization. The catalog detects a resource either because it has an Agent installed on it, or because a cloud integration is configured on it. Information about the resources in your organization are shown in the Inventory and Security tabs, with two views: List and Map.
 
 **Inventory Tab**:
-The Inventory tab shows context for a resource, including team ownership and related services. It helps you proactively identify and provide missing ownership information before you need it in an incident. The Resource Catalog shows configuration information customized for each resource type. You can search resources by specific configuration attributes such as `instance_type` for a host, or a `version` for a database.
+The Inventory tab shows context for a resource, including team ownership and related services. It helps you proactively identify and provide missing ownership information before you need it in an incident. The Resource Catalog also shows resource attributes customized for each resource type. You can search resources by specific attributes such as the instance type for a host, or the version for a database.
 
 **Security Tab**:
-The security tab allows you to gain a clear understanding of resources with security risk. By viewing misconfigurations and threats associated with resources, you can address the issue without needing to spend time and effort to gather security context. 
+The Security tab provides a security centric view of your infrastructure and allows you to understand how your resources are impacted by security risks. 
+
+- [**Misconfigurations**][10]: Tracks the security hygiene and compliance posture of your production environment, automates audit evidence collection, and enables you to remediate misconfigurations that leave your organization vulnerable to attacks.
+
+- [**Signals**][11]: Monitors file, network, and process activity across your environment to detect real-time threats to your infrastructure.
+
+- [**Identity Risks**][12]: Provides in-depth visibility into your organization’s AWS IAM risks and enables you to detect and resolve identity risks on an ongoing basis.
+
+- [**Identity Risks**][13]: Leverages infrastructure observability to detect, prioritize, and manage vulnerabilities in your organization’s containers and hosts.
```

Comment:
Both this bullet and the above say "Identity Risks" -- should this say "Vulnerabilities"? Just wanted to verify.

Reply:
yes you are correct, I need to change that